src/meta/processors/admin/CreateBackupProcessor.cpp

```diff
@@ -145,7 +145,7 @@ void CreateBackupProcessor::process(const cpp2::CreateBackupReq& req) {
     handleErrorCode(ret);
     ret = Snapshot::instance(kvstore_, client_)->blockingWrites(SignType::BLOCK_OFF);
     if (ret != nebula::cpp2::ErrorCode::SUCCEEDED) {
-      LOG(INFO) << "Cancel write blocking error";
+      LOG(INFO) << "Cancel write blocking error:" << apache::thrift::util::enumNameSafe(ret);
     }
```

Comment:
I have a question here: it has nothing to do with this pr modification.
When line 146 was executed successfully. Then core, the machine will never be able to write. . . .

Reply:
You are right, maybe I should add some logic to check this when restart after encounter core